src/cuga/sdk.py

```
@@ -1134,7 +1227,188 @@ async def sync_from_filesystem(self) -> Dict[str, Any]:
+        manager = ToolGuardManager(self._agent)
+        await manager.initialize()
+        
+        # Generate guard code using the manager
+        guard_code = await manager.generate_guard_code(
+            policy=policy,
+            target_tool=target_tool,
+            app_name=app_name
+        )
+        
+        return guard_code
+        return violating_examples, compliance_examples
```

Ruff (0.15.12)

[error] Undefined name `violating_examples`

(F821)